integration_tests/integration/homepage/homepage.cy.js

```diff
@@ -9,6 +9,7 @@ context('Homepage', () => {
       { roleCode: 'REMOVE_RESTRICTED_PATIENT' },
       { roleCode: 'TRANSFER_RESTRICTED_PATIENT' },
       { roleCode: 'SEARCH_RESTRICTED_PATIENT' },
+      { roleCode: 'RESTRICTED_PATIENT_MIGRATION' },
```

Comment:
note the new format of role code - prefixed with the service.  This is the standard way that all our services are created, but not about to fix the restricted patient role codes now.